test/browser/samples/renormalizes-external-paths/_config.js

```diff
@@ -0,0 +1,42 @@
+const { join, dirname } = require('path');
+
+module.exports = {
+	description: 'renormalizes external paths if possible',
+	options: {
+		input: ['/main.js', '/nested/entry.js'],
+		external(id) {
+			return id.endsWith('ext');
+		},
+		plugins: {
+			resolveId(source, importer) {
+				if (source.endsWith('ext.js')) {
+					return false;
+				}
+				if (!importer) {
+					return source;
+				}
+				return join(dirname(importer), source);
+			},
+			load(id) {
+				switch (id) {
+					case '/main.js':
+						return `import './nested/dep.js';
+import './ext.js';
+import './nested/nested-ext.js';`;
+					case '/dep.js':
+						return `import './ext.js';
+import './nested/nested-ext.js';`;
+					case '/nested/dep.js':
```

Comment:
Ci failing with:

```console
1 failing

  1) browser
       renormalizes-external-paths: renormalizes external paths if possible:
     Error: Could not load \nested\dep.js (imported by main.js): Unexpected id \nested\dep.js
      at Object.load (test\browser\samples\renormalizes-external-paths\_config.js:37:13)
      at D:\a\rollup\rollup\dist\rollup.browser.js:11:340192
```

I suspect in normalizing the paths on Windows, `/` -> `\` and these case statements won't work. May need to consider using `join()` or `sep` to ensure cross-OS compatibility (assuming this isn't just a bug and you're expecting the ids to have posix like paths?)

Reply:
I think the bug is using native `path` in the test config. I will fix it!